packages/mds-policy-author/api.ts

```diff
@@ -188,6 +189,10 @@ function api(app: express.Express): express.Express {
 
   app.post(pathsFor('/policies'), async (req, res) => {
     const policy = req.body
```

Comment:
You can get rid of the `parm-reassign` and handle the default with something like this:
```
const policy = { policy_id: uuid(), ...req.body }
```

Reply:
Cool, I didn't know you could do that.